docs/DeveloperGuide.md

```diff
@@ -521,330 +523,322 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
-* 1b. The given argument has missing fields.
+* 1b. The given command has missing inputs.
 
     * 1b1. SudoHR shows an error message.
 
       Use case resumes at step 1.
 
+* 1c. The employee has already taken leave on the date specified
+    * 1c1. SudoHR shows an error message.
+
+      Use case resumes at step 1.
 
+* 1d. There is no employee with the eid given as input.
+    * 1d1. SudoHR shows an error message.
 
+      Use case resumes at step 1.
 
-**Use case: UC11 - Delete event**
+**Use case: UC7 - Add employee's leave in range**
 **MSS:**
-1. User request to __list events(UC6)__
-2. SudoHR shows all events
-3. User requests to delete event from SudoHR
-4. SudoHR deletes event.
-5. Use case ends.
+1. User requests SudoHR to add leaves for a employee with a specific eid from a start date till end date inclusive.
+2. SudoHR informs user that all the employee's leaves has been successfully added.
+
+    Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The given command has invalid inputs.
 
-  Use case ends.
+   * 1a1. SudoHR shows an error message.
 
-* 3a. The given index is invalid.
+     Use case resumes at step 1.
 
-    * 3a1. SudoHR shows an error message.
+* 1b. The given command has missing inputs.
 
-      Use case resumes at step 2.
+    * 1b1. SudoHR shows an error message.
 
+      Use case resumes at step 1.
 
-**Use case: UC12 - Update event**
-**MSS:**
-1. User requests to __list events(UC6)__
-2. SudoHR shows all events and their details
-3. User requests to update event details on SudoHR
-4. SudoHR update event.
-5. Use case ends.
+* 1c. The employee has already taken leave on any of the dates in the range of the dates.
+    * 1c1. SudoHR shows an error message.
 
-**Extensions**
+      Use case resumes at step 1.
 
-* 2a. The list is empty.
+* 1d. The end date is more than 6 days away from the start date.
+    
+    * 1d1. SudoHR shows an error message.
 
-  Use case ends.
+      Use case resumes at step 1.
 
-* 3a. The given index is invalid.
+* 1e. There is no employee with the eid given as input.
+   
+    * 1e1. SudoHR shows an error message.
 
-    * 3a1. SudoHR shows an error message.
+      Use case resumes at step 1.
 
-      Use case resumes at step 2.
+* 1f. The end date is before start date.
+    
+    * 1f1. SudoHR shows an error message.
 
-* 3b. The given argument is empty.
+      Use case resumes at step 1.
+    
 
-    * 3b1. SudoHR shows an error message.
+**Use case: UC8 - Delete employee's leave**
+**MSS:**
+1. User __lists all leaves taken by employee(UC 11).__
+2. User requests to delete leave from SudoHR for the employee on a specific day.
+3. SudoHR informs user that employee's leave has been successfully deleted.
 
-      Use case resumes at step 2.
+    Use case ends.
 
+**Extensions**
 
-**Use case: UC13 - List event**
-**MSS:**
-1. User requests SudoHR to list all events
-2. SudoHR shows all events and their details
+* 1a. The list is empty.
 
+      Use case ends.
```

Comment:
The format is weird for this part
<img width="395" alt="Screenshot 2023-04-08 at 5 35 33 PM" src="https://user-images.githubusercontent.com/59330153/230714487-1665045b-88f4-4bd7-9f1a-f95e943b03d8.png">